src/EventExport/Format/TabularData/TabularDataEventFormatter.php

```diff
@@ -94,8 +94,6 @@ protected function includedOrDefaultProperties($include)
                 $properties[] = "address.addressLocality";
                 $properties[] = "address.addressCountry";
             }
-
-            array_unshift($properties, 'id');
```

Comment:
Why was this removed?

The id was always added implicitly, this was on purpose.